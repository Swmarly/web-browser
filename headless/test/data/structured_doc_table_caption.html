<!DOCTYPE html>
<html lang="en">
<body>
<table>
  <tr>
    <td>Cell 1</td>
  </tr>
  <caption>Caption should become first child of Table</caption>
  <tr>
    <td>Cell 2</td>
  </tr>
</table>
</body>
</html>
