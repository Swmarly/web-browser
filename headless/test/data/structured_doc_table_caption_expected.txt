{
   "lang": "en",
   "type": "Document",
   "~children": [ {
      "type": "Table",
      "~children": [ {
         "type": "Caption",
         "~children": [ {
            "type": "NonStruct"
         } ]
      }, {
         "type": "TR",
         "~children": [ {
            "attributes": [ {
               "Headers": [  ],
               "O": "Table"
            }, {
               "O": "Table",
               "RowSpan": 1.0
            }, {
               "ColSpan": 1.0,
               "O": "Table"
            } ],
            "type": "TD",
            "~children": [ {
               "type": "NonStruct"
            } ]
         } ]
      }, {
         "type": "TR",
         "~children": [ {
            "attributes": [ {
               "Headers": [  ],
               "O": "Table"
            }, {
               "O": "Table",
               "RowSpan": 1.0
            }, {
               "ColSpan": 1.0,
               "O": "Table"
            } ],
            "type": "TD",
            "~children": [ {
               "type": "NonStruct"
            } ]
         } ]
      } ]
   } ]
}
